libethcore/BasicAuthority.h

```diff
@@ -23,6 +23,7 @@
 
 #pragma once
 
+#include <libdevcrypto/Common.h>
```

Comment:
Why is this added now?

Reply:
Previously this header was implicitly included for Address. Now we still need it for crypto::Signature.